client/my-sites/activity/activity-log-confirm-dialog/index.jsx

```diff
@@ -52,6 +57,34 @@ const ActivityLogConfirmDialog = ( {
 						{ confirmTitle }
 					</Button>
 				</div>
+				{ config.isEnabled( 'rewind/partial-restores' ) && (
+					<div>
```

Comment:
If we use the [`<BulkSelect/>`](https://wpcalypso.wordpress.com/devdocs/design/bulk-selects) component here we should get styling for free.